### PR TITLE
Release/v3

### PR DESCRIPTION
**Pull request by noahmalmed**
Changes put in for the latest release of the app.

# Review Checklist

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

File: src/components/Markings/components/ShapeInstructionsView.js [hunk 1]
```diff
@@ -19,15 +19,27 @@ const ShapeInstructionsView = (props) => {
...
+    const ofText = (exists(min) || exists(max)) ? ' of' : ''
+    return `${numberDrawn}` + ofText + (exists(min) ? ` ${min} required` : '')
+}
+
+const maxText = (min, max) => {
+    const comma = (min !== null && !isNaN(min) && max) ? ', ' : ' '
+    const maxText = max ? `${max} maximum ` : ''
+    return comma + maxText + 'drawn'
+}
+
```
Comment (wgranger) [inline src/components/Markings/components/ShapeInstructionsView.js]: This logic is great

**Comment by wgranger** [comment]
Logic looks great on the min/max drawn; however I'm able to draw past the maximum on the test project. 
screen shot 2018-12-20 at 9 41 54 am

**Comment by noahmalmed** [comment]
Should be fixed now

**Comment by wgranger** [comment]
Looks good

**Pull request merged**